server/routes/routes.js

```diff
@@ -1,9 +1,11 @@
 const stories = require('../controllers/storyController')
 const users = require('../controllers/userController')
+const gif = require('../controllers/giphyController')
```

Comment:
Let's use a name other than gif. It will control the whole charades game most likely